news/2025/12/05/ModelingToolkitv11.md

```diff
@@ -0,0 +1,273 @@
+"core" usage of ModelingToolkit as a compiler, where the direct interface has always been more flexible for
+many use cases like building libraries and things which are not related specifically to acausal modeling of
+DAEs (for example, `@mtkmodel` still does not even support SDEs). This lets the ModelingToolkit.jl team focus
+on what ModelingToolkit does best, namely, it's a symbolic representation for models and a compiler to fast
+and stable numerical simulations. 
+
+That does not mean that we do not like DSLs on ModelingToolkit, oh not at all! Instead this is making `@mtkmodel`
+no longer a privledged DSL of the project. For example, SymBoltz.jl and Catalyst.jl are two great DSLs built on
+ModelingToolkit.jl, just as separate packages. We plan to spawn out `@mtkmodel` to a separate DSL package of
+ModelingToolkit.jl in a similar fashion, though with the code base currently having no maintainers we will 
+deprecate the package unless new maintainers come to the revive the project. This is thus an open call for new
+folks to come in.
+
+We also must note that if someone really does need a fully developed DSL that is Modelica-like and compiles to
+ModelingToolkit, but is a more fully featured version (for example, has syntax highlighting, gives syntactic
+compiler errors, supports units, is well-maintained, etc.) then we suggest checking out 
+[Dyad](https://help.juliahub.com/dyad/dev/) which is free for non-commerical use and provides exactly this DSL.
+Our intention here is not to push out `@mtkmodel` to Dyad, but instead to note how the ModelingToolkit
+project has evolved, where the open source aspect has really evolved to thrive around the symbolic compiler but
+not all of the extra language features. As such, we want to make `@mtkmodel` like every other DSL as just a
+potential add-on package, and allow users to pick from the bevy of cool DSLs built on ModelingToolkit, keeping
+all of the difficulties of developing a language separate from the largely mathematical work of ModelingToolkit.
+
+## Final Note
+
+I just want to end this by saying, I have dedicated almost every day, night, and weekend towards making the Julia
+open source community thrive. Sometimes hard decisions have to be made, and I think it is clear to everyone that
+this is one of them. It took us almost 3 years to find this solution and I believe that this version both maximizes
+the amount of open source-ness in the project, gives people ways to opt out of any GPL-ness, while also giving a
+sustainable model for the project's future. I hope the complete honesty and transparency of these changes helps
+everyone understand the reasoning behind the changes. And I, Chris Rackauckas, am available to talk about not
+just the technical aspects but also the community and funding aspects of ModelingToolkit, SciML, and the rest of
+the Julia ecosystem at any time. If you have any questions, feel free to reach out. Let's see if this makes the
```

Comment:
> Let's see if this makes the best version of the open source ecosystem.

Is this the best way to end such a blog post? It sounds a bit of a downer to my kiwi-lang. I'd either remove and end with "reach out", or something more positive.